components/Chat/Chat.vue

```diff
@@ -1096,28 +759,8 @@
     async fetch () {
       // Timeout to prevent SSR from locking up
       const timeout = process.server ? process.env.SSR_TIMEOUT : 0;
-
-      const getChatHydration = async ( channel ) => {
-        try {
-          const global = this.$store.state[Chat.namespace][Chat.$states.global];
-          if ( global === null ) return null;
-          const { data } = await this.$axios.getSSR( `https://chat.bitwave.tv/v1/messages${ global ? '' : `/${channel}` }`, { timeout } );
-          if ( data && data.success ) return data.data;
-          return [];
-        } catch ( error ) {
-          console.log( `Chat hydration request failed` );
-          console.error( error.message );
-        }
-        return null;
-      };
-
-      this.hydrationData = await getChatHydration( this.chatChannel );
-
-      if ( this.hydrationData ) {
-        await this.hydrate( this.hydrationData, true );
-      } else {
-        await this.httpHydrate();
-      }
+      // TODO: timeout
+      await this.hydrate();
```

Comment:
it may be worth keeping the timeout here over the cleaner function call since the timeout is fairly vital to preventing server from locking up when unable to get a response from the chat API. This was 100% the case for `asyncData`, but this behavior may have been improved for the fetch command.